desktop.ini

```diff
@@ -0,0 +1,2 @@
+[LocalizedFileNames]
```

Comment:
What is this file for?

Reply:
I'm not sure what it is for but I'm not the originator, its always been there